RELEASE

```diff
@@ -5,6 +5,75 @@
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
       /_/   /_/
 
+AppScale version 3.5.0, released March 2018
+Highlights of features/bugs in this release:
+
+ - Add a datastore viewer to the dashboard
+ - Improve lb responsiveness
+ - Transaction groomer
+ - Add GQL query to datastore viewer
+ - Service statistics
+ - Taskqueue service stats
+ - Add push queue stats to Hermes
+ - Don't allow editing entities with datastore viewer
```

Comment:
` - Don't allow editing entities with datastore viewer` is part of ` - Add a datastore viewer to the dashboard` and can be removed.